src/state/updaters/CollateralYieldUpdater.tsx

```diff
@@ -101,7 +102,9 @@ const CollateralYieldUpdater = () => {
     if (data?.data) {
       const yields: CollateralYieldByChain = {
         [ChainId.Mainnet]: {},
-        [ChainId.Base]: {},
+        [ChainId.Base]: {
+          aeroUSDCeUSD: 10, // TODO: hardcoded
```

Comment:
https://defillama.com/yields/pool/212375c8-694c-4fa5-8b36-05f50c8e61b2
(Also add it to the Cloudflare worker)